src/ListActions/ListActions.js

```diff
@@ -555,267 +285,33 @@ var ListActions = module.exports = kind(
 	},
 
 	/**
-	* Toggles value of `this.open`.
-	*
-	* @private
-	*/
-	expandContract: function (sender, e) {
-		if (this.disabled) {
-			return true;
-		}
-		var open = !this.getOpen();
-		if (open) {
-			this.doShow();
-		} else {
-			this.doHide();
-		}
-		this.setOpen(open);
-	},
-
-	/**
-	* @private
-	*/
-	beforeOpenDrawer: function (standardHeight, type) {
-		this.standardHeight = standardHeight;
-	},
-
-	//TODO: Remove the onListActionOpenChanged event. It will be deprecated in favor of the onShow/onHide events
-	// once we communicate to SmartShare (the only app we could find that's handling this event).
-	/**
-	* @fires module:moonstone/ListActions~ListActions#onListActionOpenChanged
-	* @private
-	*/
-	openChanged: function () {
-		this.$.drawer.set('spotlightDisabled', !this.getOpen());
-		this.setActive(this.getOpen());
-		this.doListActionOpenChanged({open: this.open});
-
-		if (this.allowBackKey) {
-			if (this.open) this.pushBackHistory();
-			else if (!EnyoHistory.isProcessing()) EnyoHistory.drop();
-		}
-
-		// If opened, show drawer and resize it if needed
-		if (this.open) {
-			if (this.drawerNeedsResize) {
-				this.resizeDrawer();
-				this.drawerNeedsResize = false;
-			}
-			// Capture onSpotlightFocus happening outside the drawer, so that we can prevent focus
-			// from landing in the header beneath the drawer
-			dispatcher.capture(this.$.drawer, {onSpotlightFocus: 'capturedSpotlightFocus'}, this);
-		} else {
-			dispatcher.release(this.$.drawer);
-		}
-	},
-
-	/**
-	* @fires module:moonstone/TooltipDecorator~TooltipDecorator#onRequestMuteTooltip
-	* @fires module:moonstone/TooltipDecorator~TooltipDecorator#onRequestUnmuteTooltip
+	* @override
 	* @private
 	*/
-	drawerAnimationEnd: function (sender, event) {
-		var rendered = event && event.rendered;
-
-		//on closed, hide drawer and spot _this.$.activator_
-		if (!this.getOpen()) {
-			this.drawerClosed(rendered);
-		}
-		//on open, move top and spot _this.$.closeButton_
-		else {
-			this.drawerOpened(rendered);
-		}
-		return true;
-	},
-
-	/**
-	* @private
-	*/
-	drawerClosed: function (rendered) {
-		this.bubble('onRequestUnmuteTooltip');
-		if (this.generated && !rendered) {
-			Spotlight.spot(this.$.activator);
-		}
-
-		if (!rendered) this.doHidden();
-	},
-
-	/**
-	* @private
-	*/
-	drawerOpened: function (rendered) {
-		if (this.resetScroller) {
-			this.$.listActions.scrollTo(0, 0);
-			this.resetScroller = false;
-		}
-		if (this.generated && !rendered) {
-			Spotlight.spot(this.$.closeButton);
-		}
+	popupShown: function() {
+		ContextualPopupDecorator.prototype.popupShown.apply(this,arguments);
 		this.bubble('onRequestMuteTooltip');
```

Comment:
Yes I believe it's still needed. `ListActions` which is a subkind of `ContextualPopupDecorator` can be also wrapped in `TooltipDecorator`. If we don't mute tooltip, every spotlight focus inside the `ListActionsPopup` will show tooltip. It may look alright, but spotlight blur will disappear tooltip, which means that every spot move in both 5-way and pointer mode will cause to hide a show tooltip with delays. I don't think that would be okay.